src/types/language.ts

```diff
@@ -0,0 +1 @@
+export type Language = 'de' | 'fr' | 'it' | 'en';
```

Comment:
Not really part of this PR but I wonder if it would make sense to create a type package since this one needs to be shared across a lot of projects and packages. WDYT?

Reply:
It would already be part of our i18n package. If we decide to integrate that, we could replace it